lib/taskana-core/src/main/java/pro/taskana/task/internal/TaskQueryImpl.java

```diff
@@ -68,7 +68,8 @@ public class TaskQueryImpl implements TaskQuery {
   private boolean addAttachmentClassificationNameToSelectClauseForOrdering = false;
   private boolean addWorkbasketNameToSelectClauseForOrdering = false;
   private boolean joinWithUserInfo;
-
+  private boolean groupByPor;
```

Comment:
This field is only used in TaskQuerySqlProvider. There, the variable name is specified as a String so that MyBatis can access it. However, Sonarcloud sees it as "unused", because it doesn't recognize the reference to the variable because it's in a String. How can I fix it?

Reply:
OK, didn't realize that it is used by MyBatis...


Can you try what sonarcloud says when you add this annotation:
`@SuppressWarnings("unused")`

If this doesn't help, we ignore this code smell

Reply:
We don't have this annotation for other fields, even the "unused" ones. It would be more consistent not to add it. What do you think about it?